File: ej03_imc/typescript/solucion.ts
type CategoriaImc = "Bajo peso" | "Peso normal" | "Sobrepeso" | "Obesidad"

export const clasificarImc = (peso: number, altura: number): CategoriaImc => {
    const imc = peso / altura ** 2

    if (imc < 18.5) return "Bajo peso"
    else if (imc < 25) return "Peso normal"
    else if (imc < 30) return "Sobrepeso"
    else return "Obesidad"
}

File: ej03_imc/python/solucion.py
def clasificar_imc(peso: float, altura: float) -> str:
    imc = peso / altura**2

    if imc < 18.5:
        return "Bajo peso"
    elif imc < 25:
        return "Peso normal"
    elif imc < 30:
        return "Sobrepeso"
    else:
        return "Obesidad"
